edx_rest_framework_extensions/tests/test_jwt_decoder.py

```diff
@@ -49,6 +49,8 @@ def generate_jwt_payload(user, scopes=None, filters=None, version='1.0.0'):
         payload['scopes'] = scopes
     if filters:
         payload['filters'] = filters
+    if is_restricted is not None:
+        payload['is_restricted'] = is_restricted
```

Comment:
Why is `is_restricted` not just boolean?

Reply:
Since we didn't always include the `is_restricted` boolean, the `None` value allows it to be conditionally included.